.github/validator/validator.py

```diff
@@ -0,0 +1,280 @@
+import os
+import argparse
+import subprocess
+import logging
+
+azure_dev_workflow_file_path = ".github/workflows/azure-dev.yml"
```

Comment:
We should move this to a published github action that will be easier to reuse from multiple templates/projects.

Reply:
I agree with that too. It's my next step to move validator.py as a published github action. But it may need more process for release.